lib/services.js

```diff
@@ -252,8 +225,11 @@ export default {
                     log.warn('error parsing acl', { error: err });
                     return cb(err);
                 }
-                omVal.acl = parsedACL;
-                metadata.putObjectMD(bucketName, objectKey, omVal, log, err => {
+                md.setAcl(parsedACL);
+                log.trace('object metadata', {
+                    omVal: md.getData(),
```

Comment:
im okay to remove it, but its not my goal, since it can be useful for s3 debugging @LaurenSpiegel ?